plugins/by-name/snacks/default.nix

```
@@ -0,0 +1,205 @@
+{ lib, ... }:
+let
+  inherit (lib) types;
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "snacks";
+  package = "snacks-nvim";
+
+  maintainers = [ lib.maintainers.HeitorAugustoLN ];
+
+  settingsOptions = {
+    bigfile = {
+      enabled = defaultNullOpts.mkBool true ''
+        Whether to enable `bigfile` plugin. 
+      '';
+      notify = defaultNullOpts.mkBool true ''
```

```suggestion
      '';

      notify = defaultNullOpts.mkBool true ''
```
Please, add a blank line between each option declaration.